File: IG2/20220913/StringIndex.py
string = 'Computer Science'

print("Original string: ", string)


def tryit(text, i):
    try:
        a = text[i]
    except IndexError:
        return IndexError
    return a


print("\nstring[0]: \n", tryit(string, 0))
print("\nstring[1]: \n", tryit(string, 1))
print("\nstring[15]: \n", tryit(string, 15))
print("\nstring[16]: \n", tryit(string, 16))
print("\nstring[-1]: \n", tryit(string, -1))
print("\nstring[100]: \n", tryit(string, 100))
print("\nstring[-100]: \n", tryit(string, -100))

File: IG2/20220913/StringIndex.java
class StringIndex {
	public static void main(String[] args) {
		String string = "Computer Science";

		System.out.println("Original string: " + string);

		System.out.println("\nstring.charAt(0): \n" + tryit(string, 0));
		System.out.println("\nstring.charAt(1): \n" + tryit(string, 1));
		System.out.println("\nstring.charAt(15): \n" + tryit(string, 15));
		System.out.println("\nstring.charAt(16): \n" + tryit(string, 16));
		System.out.println("\nstring.charAt(-1): \n" + tryit(string, -1));
		System.out.println("\nstring.charAt(100): \n" + tryit(string, 100));
		System.out.println("\nstring.charAt(-100): \n" + tryit(string, -100));
	}

	public static String tryit(String text, int i) {
		String a;
		try {
			a = String.valueOf(text.charAt(i));
		} catch (Exception e) {
			a = String.valueOf(e);
		}
		return a;
	}
}